exporter/clickhouseexporter/config.go

```diff
@@ -142,3 +153,26 @@ func (cfg *Config) buildDB(database string) (*sql.DB, error) {
 	return conn, nil
 
 }
+
+// Generate ENGINE string
```

Comment:
Comments should begin with the name of the thing being described and end in a period. https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences